packages/components/src/components/search/search.tsx

```diff
@@ -188,11 +203,19 @@ export class Search implements ComponentInterface {
     private search(inputValue): void {
         Promise.all([
             this.makeMapsIndoorsQuery(inputValue),
-            this.makeGooglePlacesQuery(inputValue)
+            this.makeGooglePlacesQuery(inputValue),
+            this.getMapboxSearchResults(inputValue),
         ])
             .then(results => {
                 this.lastRequested = inputValue;
-                this.pushResults(results[0].concat(results[1]));
+
+                if (this.google) {
+                    this.pushResults(results[0].concat(results[1]));
+                } else if (this.mapbox && mapboxgl.accessToken) {
+                    this.pushResults(results[0].concat(results[2]));
+                } else {
+                    this.pushResults(results[0]);
+                }
```

Comment:
What happens if an insane developer provides both Google and Mapbox props? 😱

Reply:
Hmmm, right now if we provide both Mapbox and Google props as true on the search field, we get an error because the Google is not defined.
![image](https://github.com/MapsPeople/web-ui/assets/113503166/db450f0e-771a-4daa-8bf0-c24baf5f53a1)

Reply:
Yeah, it's probably a scenario that will never happen. I guess we can dismiss it for now.